setup_vkd3d.sh

```diff
@@ -0,0 +1,191 @@
+#!/bin/bash
+
+# default directories
+vkd3d_lib32=${vkd3d_lib32:-"x86"}
+vkd3d_lib64=${vkd3d_lib64:-"x64"}
+
+# figure out where we are
+basedir=$(dirname "$(readlink -f $0)")
+
+# figure out which action to perform
+action="$1"
+
+case "$action" in
+install)
+  ;;
+uninstall)
+  ;;
+*)
+  echo "Unrecognized action: $action"
+  echo "Usage: $0 [install|uninstall] [--symlink]"
+  exit 1
+esac
+
+# process arguments
+shift
+
+file_cmd="cp -v"
+
+while (($# > 0)); do
+  case "$1" in
+  "--symlink")
+    file_cmd="ln -s -v"
+    ;;
+  esac
+  shift
+done
+
+# check wine prefix before invoking wine, so that we
+# don't accidentally create one if the user screws up
+if [ -n "$WINEPREFIX" ] && ! [ -f "$WINEPREFIX/system.reg" ]; then
+  echo "$WINEPREFIX:"' Not a valid wine prefix.' >&2
+  exit 1
+fi
+
+# find wine executable
+export WINEDEBUG=-all
+
+wine="wine"
+wine64="wine64"
+wineboot="wineboot"
+
+# $PATH is the way for user to control where wine is located (including custom Wine versions).
+# Pure 64-bit Wine (non Wow64) requries skipping 32-bit steps.
+# In such case, wine64 and winebooot will be present, but wine binary will be missing,
+# however it can be present in other PATHs, so it shouldn't be used, to avoid versions mixing.
+wine_path=$(dirname "$(which $wineboot)")
+wow64=true
+if ! [ -f "$wine_path/$wine" ]; then
+   wine=$wine64
+   wow64=false
+fi
+
+# resolve 32-bit and 64-bit system32 path
+winever=$($wine --version | grep wine)
+if [ -z "$winever" ]; then
+    echo "$wine:"' Not a wine executable. Check your $wine.' >&2
+    exit 1
+fi
+
+# ensure wine placeholder dlls are recreated
+# if they are missing
+$wineboot -u
+
+win64_sys_path=$($wine64 winepath -u 'C:\windows\system32' 2> /dev/null)
+win64_sys_path="${win64_sys_path/$'\r'/}"
+if $wow64; then
+  win32_sys_path=$($wine winepath -u 'C:\windows\system32' 2> /dev/null)
+  win32_sys_path="${win32_sys_path/$'\r'/}"
+fi
+
+if [ -z "$win32_sys_path" ] && [ -z "$win64_sys_path" ]; then
+  echo 'Failed to resolve C:\windows\system32.' >&2
+  exit 1
+fi
+
+# create native dll override
+overrideDll() {
+  $wine reg add 'HKEY_CURRENT_USER\Software\Wine\DllOverrides' /v $1 /d native /f >/dev/null 2>&1
+  if [ $? -ne 0 ]; then
+    echo -e "Failed to add override for $1"
+    exit 1
+  fi
+}
+
+# remove dll override
+restoreDll() {
+  $wine reg delete 'HKEY_CURRENT_USER\Software\Wine\DllOverrides' /v $1 /f > /dev/null 2>&1
+  if [ $? -ne 0 ]; then
+    echo "Failed to remove override for $1"
+  fi
+}
+
+# copy or link vkd3d dll, back up original file
+installFile() {
+  dstfile="${1}/${3}.dll"
+  srcfile="${basedir}/${2}/${3}.dll"
+
+  if [ -f "${srcfile}.so" ]; then
+    srcfile="${srcfile}.so"
+  fi
```

Comment:
I remember this to be a fallback path for DXVK `.dll.so` builds, but afaik vkd3d doesn't produce any such files, making this unnecessary and safe to remove (especially since this script is installed only when building `d3d12.dll`).
```suggestion
```